pymc3/model.py

```diff
@@ -458,7 +458,7 @@ def cont_vars(self):
         """All the continuous variables in the model"""
         return list(typefilter(self.vars, continuous_types))
 
-    def Var(self, name, dist, data=None):
+    def Var(self, name, dist, data=None, population=None):
```

Comment:
I like both `population` and `total_size`. `total size` is probably more intuitive